app/views/businesses/show.html.erb

```diff
@@ -3,11 +3,9 @@
   <%= @business.description %>
 </div>
 
-<% if current_user %>
-  <% if current_user.admin? || current_user.platform_admin? %>
+<% if current_admin? || platform_admin? %>
```

Comment:
`current_admin?` includes the OR check. It returns true for Business Admins or Platform Admins